test/test_um2netcdf.py

```diff
@@ -217,3 +220,121 @@ def test_cube_filtering_no_include_exclude(ua_plev_cube, heaviside_uv_cube):
+    fake_cube.cell_methods = []
+    return fake_cube
+
+
+UMStash = namedtuple("UMStash",
+                     "long_name, name, units, standard_name, uniquename")
+
+
+@pytest.fixture
+def um_var_empty_std():
+    """Return an empty um_var lookup."""
+    um_var = UMStash("", "", "", "", "")
+    return um_var
+
+
+CellMethod = namedtuple("CellMethod", "method")
+
+
+@pytest.fixture
+def max_cell_method():
+    return CellMethod("maximum")
+
+
+@pytest.fixture
+def min_cell_method():
+    return CellMethod("minimum")
+
+
+def test_rename_cube_var_name_simple(x_wind_cube, um_var_empty_std):
+    # NB: ignores cell methods functionality
+    assert x_wind_cube.var_name == "var_name"  # dummy initial value
+    um2nc.rename_cube_var_name(x_wind_cube, None, simple=True)
+    assert x_wind_cube.var_name == "fld_s00i002"
+
+
+def test_rename_cube_var_rename_with_cell_methods_max(x_wind_cube,
+                                                      um_var_empty_std,
+                                                      max_cell_method):
+    x_wind_cube.cell_methods = [max_cell_method]
+
+    um2nc.rename_cube_var_name(x_wind_cube, None, simple=True)
+    assert x_wind_cube.var_name == "fld_s00i002_max"
+
+
+def test_rename_cube_var_rename_with_cell_methods_min(x_wind_cube,
+                                                      um_var_empty_std,
+                                                      min_cell_method):
+    x_wind_cube.cell_methods = [min_cell_method]
+
+    um2nc.rename_cube_var_name(x_wind_cube, None, simple=True)
+    assert x_wind_cube.var_name == "fld_s00i002_min"
+
+
+def test_rename_cube_var_name_unique(x_wind_cube):
+    # NB: ignores cell methods functionality
+    unique = "unique_string_name"
+    um_unique = UMStash("", "", "", "", unique)
+    um2nc.rename_cube_var_name(x_wind_cube, um_unique, simple=False)
+    assert x_wind_cube.var_name == unique
+
+
+def test_rename_cube_standard_name_x_wind(x_wind_cube, um_var_empty_std):
+    um2nc.rename_cube_names(x_wind_cube, um_var_empty_std, verbose=False)
+    assert x_wind_cube.standard_name == "eastward_wind"
+
+
+def test_rename_cube_standard_name_y_wind(um_var_empty_std):
+    fake_cube = PartialCube("var_name", {'STASH': DummyStash(0, 3)}, "y_wind")
+    fake_cube.cell_methods = []
+
+    um2nc.rename_cube_names(fake_cube, um_var_empty_std, verbose=False)
+    assert fake_cube.standard_name == "northward_wind"
+
+
+def test_standard_name_umvar_name_mismatch(x_wind_cube):
+    um_var = UMStash("", "", "", "fake", "")
+    um2nc.rename_cube_names(x_wind_cube, um_var, verbose=False)
+    assert x_wind_cube.standard_name == um_var.standard_name
+
+
+def test_standard_name_umvar_name_mismatch_warn(x_wind_cube):
+    um_var = UMStash("", "", "", "fake", "")
+
+    with pytest.warns():
+        um2nc.rename_cube_names(x_wind_cube, um_var, verbose=True)
+
+    assert x_wind_cube.standard_name == um_var.standard_name
+
+
+def test_rename_cubes_long_name(x_wind_cube):
```

Comment:
Is this test missing the `assert` right at the end?

Reply:
Yes... whoops!
It's fixed here https://github.com/ACCESS-NRI/um2nc-standalone/pull/52/commits/f9e1a3766c81b630e574583d009db9830f11719c.